indico/modules/rb_new/client/js/common/linking/LinkBar.jsx

```diff
@@ -0,0 +1,87 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2019 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+import {Icon, Popup} from 'semantic-ui-react';
+import {Translate} from 'indico/react/i18n';
+import * as linkingActions from './actions';
+import * as linkingSelectors from './selectors';
+import {linkDataShape} from './props';
+
+import './LinkBar.module.scss';
+
+
+const messages = {
+    event: Translate.string('Your booking will be linked to an event:'),
+    contribution: Translate.string('Your booking will be linked to a contribution:'),
+    sessionBlock: Translate.string('Your booking will be linked to a session block:'),
+};
+
+/**
+ * `LinkBar` shows an indicator in case the user's booking will be linked to
+ * an event, contribution or session block.
+ */
+const LinkBar = ({visible, clear, data}) => {
+    if (!visible) {
+        return null;
+    }
+
+    const {type, title, eventURL, eventTitle} = data;
+    return (
+        <header styleName="link-bar">
```

Comment:
Intentionally `header` instead of SUI's `<Header />`?

Reply:
I used the same element already used for the rb-menu-bar. I think SUI Header comes with extra styling.